.aws/task-definition-prod.json

```diff
@@ -0,0 +1,65 @@
+{
+  "containerDefinitions": [
+    {
+      "name": "tis-revalidation-concerns",
+      "image": "430723991443.dkr.ecr.eu-west-2.amazonaws.com/tis-revalidation-concerns:latest",
+      "portMappings": [
+        {
+          "containerPort": 8087
+        }
+      ],
+      "environment": [
+        {
+          "name": "TITLE",
+          "value": "tis-revalidation-concerns"
+        },
+        {
+          "name": "AWS_REGION",
+          "value": "eu-west-2"
+        },
+        {
+          "name": "SENTRY_ENVIRONMENT",
+          "value": "prod"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "awslogs-prod-tis-revalidation-concerns",
+          "awslogs-region": "eu-west-2",
+          "awslogs-stream-prefix": "awslogs-tis-revalidation-concerns"
+        }
+      },
+      "secrets": [
+        {
+          "name": "MONGODB_HOST",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-mongo-prod-host"
+        },
+        {
+          "name": "MONGODB_PORT",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-mongo-prod-port"
+        },
+        {
+          "name": "MONGODB_USERNAME",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-mongo-prod-user"
+        },
+        {
+          "name": "MONGODB_PASSWORD",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-mongo-prod-password"
+        },
+        {
+          "name": "SENTRY_DSN",
+          "valueFrom": "arn:aws:ssm:eu-west-2:430723991443:parameter/tis-revalidation-concern-sentry-dsn"
+        }
+      ]
+    }
+  ],
+  "executionRoleArn": "ecsTaskExecutionRole",
+  "family": "tis-revalidation-concerns-prod",
+  "requiresCompatibilities": [
+    "FARGATE"
+  ],
+  "networkMode": "awsvpc",
+  "cpu": "256",
+  "memory": "4096"
```

Comment:
Should be 1G

Reply:
Do we need to match it with the task definition in TIS-OPS and the ECS main.tf? They are both 4096 for memory.

https://github.com/Health-Education-England/TIS-OPS/blob/master/terraform/environments/prod/revalidation/ecs/task-definitions/concerns.json

https://github.com/Health-Education-England/TIS-OPS/blob/master/terraform/environments/prod/revalidation/ecs/main.tf

Reply:
Everything should match what is in this project's task-definition file.

Reply:
Updated to 1024. Thanks.